.mailmap

```diff
@@ -0,0 +1,19 @@
+Dongdong Tian <seisman.info@gmail.com>
+Dongdong Tian <seisman.info@gmail.com> <seisman@users.noreply.github.com>
+Jiayuan Yao <coreman.seism@gmail.com>
+Jiayuan Yao <coreman.seism@gmail.com> <core-man@users.noreply.github.com>
+Jing-Hui Tong <86273921+JingHuiTong@users.noreply.github.com>
+Jing-Hui Tong <86273921+JingHuiTong@users.noreply.github.com> <r0922422020@g.ntu.edu.tw>
+Max Jones <meghanj@alum.mit.edu>
+Max Jones <meghanj@alum.mit.edu> <14077947+maxrjones@users.noreply.github.com>
+Max Jones <meghanj@alum.mit.edu> <meghanj@hawaii.edu>
+Max Jones <meghanj@alum.mit.edu> <meghanrjones@users.noreply.github.com>
```

Comment:
Max's most recent commit at 893546c87e92792b2a40667075a002e5ea8081c6 uses his GitHub noreply email, probably should use that instead.

```suggestion
Max Jones <14077947+maxrjones@users.noreply.github.com>
Max Jones <14077947+maxrjones@users.noreply.github.com> <meghanj@alum.mit.edu>
Max Jones <14077947+maxrjones@users.noreply.github.com> <meghanj@hawaii.edu>
Max Jones <14077947+maxrjones@users.noreply.github.com> <meghanrjones@users.noreply.github.com>
```